modules/hal_nordic/nrfx/nrfx_config_reserved_resources.h

```diff
@@ -0,0 +1,710 @@
+/*
+ * Copyright (c) 2024, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFX_CONFIG_RESERVED_RESOURCES_H__
+#define NRFX_CONFIG_RESERVED_RESOURCES_H__
+
+/** @brief Bitmask that defines GPIOTE130 channels reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_GPIOTE130_CHANNELS_USED                                                               \
+	(~NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote130), owned_channels) |                           \
+	 NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote130), child_owned_channels))
+
+/** @brief Bitmask that defines GPIOTE131 channels reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_GPIOTE131_CHANNELS_USED                                                               \
+	(~NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote131), owned_channels) |                           \
+	 NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote131), child_owned_channels))
+
+/** @brief Bitmask that defines EGU instances that are reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_EGUS_USED 0
+
+/** @brief Bitmask that defines TIMER instances that are reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_TIMERS_USED 0
+
+/*
+ * The enabled Bluetooth controller subsystem is responsible for providing
+ * definitions of the BT_CTLR_USED_* symbols used below in a file named
+ * bt_ctlr_used_resources.h and for adding its location to global include
+ * paths so that the file can be included here for all Zephyr libraries that
+ * are to be built.
+ */
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+#include <bt_ctlr_used_resources.h>
```

Comment:
Please have a look here
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/softdevice_controller/include/sdc_soc.h

Shouldn't it be here
```c
#include <sdc_soc.h>
```
?
Because of that shouldn't we use macros `SDC_xxxx_CHANNELS_USED_MASK` when defining
`NRFX_xxxx_CHANNELS_USED_BY_xxxx` below ?
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/softdevice_controller/include/sdc_soc.h#L40-L43

Reply:
This won't be added in upstream.